src/game/GamePlay.as

```diff
@@ -393,6 +393,11 @@ package game
 
         private function initCore():void
         {
+            // Bound Isolation Note Mod
+            if (options.isolationOffset >= song.chart.Notes.length)
+                options.isolationOffset = song.chart.Notes.length - 1;
```

Comment:
Would `song.chart.Notes.length` be 0 in some cases? If so, the isolation offset would be negative.